<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>13</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(28, 28, 28);</string>
  </property>
  <widget class="QWidget" name="widget">
   <widget class="QLineEdit" name="inputLineEdit">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>590</y>
      <width>741</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>0xProto Nerd Font</family>
      <pointsize>13</pointsize>
      <stylestrategy>PreferDefault</stylestrategy>
      <kerning>false</kerning>
      <fontweight>Medium</fontweight>
     </font>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::ContextMenuPolicy::PreventContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 6;
background-color: rgba(88, 88, 88,88,8);
padding: 10px;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="sendButton">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>590</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>0xProto Nerd Font</family>
      <pointsize>13</pointsize>
      <bold>true</bold>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:  #2148C0;
background-color: rgb(255, 255, 255);
border-radius: 6;</string>
    </property>
    <property name="text">
     <string>SEND</string>
    </property>
   </widget>
   <widget class="QLabel" name="badapatraLabel">
    <property name="geometry">
     <rect>
      <x>468</x>
      <y>30</y>
      <width>345</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
      <bold>true</bold>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="text">
     <string>à¤ˆ - BADAPATRA</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QScrollArea" name="chatScrollArea">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1280</width>
      <height>550</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>1280</width>
      <height>550</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>0xProto Nerd Font</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Plain</enum>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarPolicy::ScrollBarAsNeeded</enum>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="chatContainerWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1280</width>
       <height>550</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>550</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>0xProto Nerd Font</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>1221</width>
        <height>511</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="chatLayout"/>
     </widget>
    </widget>
   </widget>
   <zorder>chatScrollArea</zorder>
   <zorder>inputLineEdit</zorder>
   <zorder>sendButton</zorder>
   <zorder>badapatraLabel</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
